python-toolbox/marvin_python_toolbox/management/templates/python-engine/install_automl.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+###############################################
+# Marvin AutoML installation Script           #
+###############################################
+cmd=(dialog --title "Marvin AutoML" --separate-output --checklist "Select tools:" 22 76 16)
+options=(1 "auto-sklearn" off    # any option can be set to default to "on"
+         2 "h2o AutoML" off
+         3 "TPOT" off)
+choices=$("${cmd[@]}" "${options[@]}" 2>&1 >/dev/tty)
+clear
+for choice in $choices
+do
+    case $choice in
+        1)
+            echo "Instaling auto-sklearn..."
+            wget https://raw.githubusercontent.com/automl/auto-sklearn/master/requirements.txt \
+                 | xargs -n 1 -L 1 pip install
+            pip install auto-sklearn
+            ;;
+        2)
+            echo "Installing h2o..."
```

Comment:
There is some h2o stuff in the code.
Are we sure there is no copyright issue?

Reply:
He's open source, under Apache 2.0 License. So I don't think that will be an issue.